File: AirportManagement.API/AirportManagement.API/Data/Models/Booking.cs
using System;
namespace AirportManagement.API.Models
{
	public class Booking
	{
		public int Id { get; set; }

		public string Status { get; set; }

		public string BookingPlatform { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }

		public ICollection<BaggageCheck>? BaggageChecks { get; set; }

		public ICollection<FlightManifest>? FlightManifests { get; set; }

		public ICollection<BoardingPass>? BoardingPasses { get; set; }

		public ICollection<Baggage>? Baggages { get; set; }

		public int PassangerId { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/Booking.ts
import { Baggage, BaggageForPassanger } from "./Baggage";
import { BaggageCheck, BaggageCheckForPassanger } from "./BaggageCheck";
import { BoardingPass, BoardingPassForPassanger } from "./BoardingPass";
import { FlightManifest, FlightManifestForPassanger } from "./FlightManifest";
import { Passangers } from "./Passangers";

export interface Booking {
    Id: number;
    Status: string;
    BookingPlatform: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    BaggageChecks: BaggageCheck[];
    FlightManifests: FlightManifest[];
    BoardingPass: BoardingPass[];
    Baggages: Baggage[]
    PassangerId: number;
    Passangers: Passangers;
}


export interface BookingForPassanger{
    status: string;
    bookingPlatform: string;
    createdAt: Date;
    updatedAt: Date;
    baggageChecks: BaggageCheckForPassanger[];
    flightManifests: FlightManifestForPassanger[]
    boardingPasses: BoardingPassForPassanger[]
    baggages: BaggageForPassanger[]
}
